app/nwcScanner.tsx

```diff
@@ -31,17 +43,35 @@ export default function SettingsPage() {
     } = validateNwcURI(uri);
 
     if (!isValid || !secret) {
-      toast.show("Invalid NWC");
+      toast.show("invalid NWC");
```

Comment:
Slightly more instructive
```suggestion
      toast.show("invalid NWC string, please check the contents and try again");
```